### DISCUSSION TITLE
Is there a way to use nginx proxy manager on an appliance that forces ssl and uses a self signed cert?

### DISCUSSION THREAD

**Discussion by genfoch01** [NginxProxyManager/nginx-proxy-manager#3217] [General]
I have a readynas. This appliance generates its own self signed cert and also forces SSL when you connect to it. I thought I'd fix the  "this connection is not private " warning by putting it behind Nginx proxy manager. 

unfortunately when I do this, I don't get the cert I put on nginx   I get the self signed cert on the readynas which gives me a self signed cert error "the certificate is not trusted because it is self signed" 

the nas does not allow me to turn off ssl nor does it allow me up upload a new cert. Is there a way to configure nginx to strip out and ignore the self signed cert and then use the new one I set up?

thanks,
 GF

**Comment by thunermay** [comment]
Hi,
I have the same problem with my router (Fritz!box 7590). Does the SSL certificate of the nginxproxymanager only work, when there is no SSL-certificate? My router, I think, does not force SSL but it does not work nevertheless.

**Reply by genfoch01** [reply]
for what its worth, I do believe nginx proxy manager only works if the target system does not have its own cert.  During my testing, even though I had a new cert,  my browser showed the old cert and an error about the cert not matching the site.  

I do not yet have a solution but I think it will involve removing the cert from the old site (  via squid 3.1 or better). from what I have read, squid can use "sslbump" allowing it to open the ssl and extract the results and present that to nginx proxy manager which may solve the issue but I have not had the time to look further into than that. 

based on the number of total responses since I made this post (last September), 1: that being yours, This does not appear to be a good place to get help with nginx proxy manager. If you happen to find a more responsive site post it back here if you would. 

Thanks,
GF
